tests/button/buttonview.js

```diff
@@ -99,20 +99,4 @@ describe( 'ButtonView', () => {
 			} );
 		} );
 	} );
-
-	describe( '_setupIcon', () => {
```

Comment:
Yep. It's the same as https://github.com/ckeditor/ckeditor5-ui-default/blob/d6c6f31bc0acfeabdb0f5592ed2df3269f4a5b27/tests/button/button.js#L74-L87 isn't it? It's a regression after #23.